templates/zerver/integrations/hubot.md

```diff
@@ -1,43 +1,67 @@
-1. Follow the **Getting Started with Hubot** section of the
-   [Hubot README](https://hubot.github.com/docs/#getting-started-with-hubot)
-   to create your Hubot. You'll have a new directory from
-   which `bin/hubot` starts a vanilla Hubot instance with
-   the shell backend.
+# Zulip Hubot Integration
 
-1. In your Hubot's directory, install the Zulip adapter. Run:
-   `npm install --save hubot-zulip`
+Use Hubot to execute scripts and commands within Zulip!
+
+{start_tabs}
+
+1. Follow the [**Getting Started with Hubot**][getting-started] guide
+   to create your Hubot. You'll have a new directory from which `bin/hubot`
+   starts a vanilla Hubot instance with the shell backend.
+
+1. In your Hubot's directory, install the Zulip adapter by running:
+
+      `npm install --save hubot-zulip`
 
 1. {!create-a-generic-bot.md!}
-   Note its username, API key and full name; you will use them
-   on the next step.
 
-1. To run Hubot locally, first, set the following environment
-   variables by running:
+1. Hubot uses the following environment variables, set them using the
+   information of the bot you created, by running:
 
     ```
     export HUBOT_ZULIP_SITE="{{ api_url }}"
     export HUBOT_ZULIP_BOT="hubot-bot@example.com"
-    export HUBOT_ZULIP_API_KEY="your_key"
+    export HUBOT_ZULIP_API_KEY="<your_key>"
     ```
 
-    Then, run:
+1. Run Hubot:
+
+    `bin/hubot --adapter zulip --name "<bot username>"`
 
-    `bin/hubot --adapter zulip --name "myhubot"`
+!!! tip ""
 
-    The `--name` parameter must match the name you gave the bot on
-    the settings page.
+    Hubot automatically listens for commands on all public channels. Private
+    channels require an invitation. Hubot's access can be
+    [configured](#configuration-options).
 
-Hubot will automatically listen for commands on all public streams.
-You can also invite Hubot to private streams.
+{end_tabs}
 
 To test your Hubot installation, send it an @-notification with a
-basic command, for example `@Hubot pug me`, which should produce a
+basic command, e.g., `@Hubot pug me`, which should produce a
 result like this:
 
 ![Hubot message](/static/images/integrations/hubot/001.png)
```

Comment:
It should be pretty easy to just fake this interaction to make a screenshot. One can do it manually by just making a bot with that avatar and having it reply using `zulip-send`. 

It might not be hard to extend `tools/screenshots/generate-user-messages-screenshot` to do that automatically, though. Maybe you can chat with @roanster007; he may be able to whip up an extension of it that does this (takes a bot avatar and a configured message body and makes a screenshot for that bot having sent that message).